docs/_Research-Data-Management/13-data-qc.md

```diff
@@ -1,196 +1,387 @@
----
-title: Data Quality Control
-category: Research-Data-Management
-layout: default
-docs_css: markdown
----
 
-Qualtiy Control Expert Q&A
+The following data compiled from experts of NFDI4Microbiota members.
+
+The goal is to gather this information centrally in NFDI4Microbiota to assess the span of tools and derive consortium-wide guidelines for the tightest integration possible.
```

Comment:
Maybe add a few follow-up sentences that the following chunks/sections will start with a statement of the issue/problem (such as "Library contamination or highly homogenous library" or "Extremely low read counts (Mapping)") with sometimes the category indicated in brackets. Possible reasons for the situation/issue/problem are then stated and when known a solution is indicated via a :bulb: emoji.

The phrasing here is probably not great, but it's to have a guide at the start so that readers knows the structure and what to expect.